app/src/main/res/layout/fragment_upload_media_detail_fragment.xml

```diff
@@ -6,11 +6,12 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <com.github.chrisbanes.photoview.PhotoView
```

Comment:
I believe this was the last usage of PhotoView, we could remove the dependency

Reply:
@macgills Fair point removed this